<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreManageDialog</class>
 <widget class="QDialog" name="StoreManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage local repository</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topLayout" stretch="0,4,2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="versionLabel">
       <property name="text">
        <string>Command</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="commandCombo"/>
     </item>
     <item>
      <widget class="QComboBox" name="versionCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="log">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspc">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>388</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="actionButton">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
